src/components/singular/JobGallery.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import investorPhotos from "../../assets/images/investors/InvestorPhotos";
+import {
+  BsFillArrowRightCircleFill,
+  BsCaretDownFill,
+  BsFillCaretUpFill,
+} from "react-icons/bs";
+
+const JobGallery = () => {
+  const jobs = [
+    {
+      name: "Warren Buffett",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "50",
+      id: "investor",
+    },
+    {
+      name: "Money Maker",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "50",
+      id: "bot",
+    },
+    {
+      name: "Jordan Belfort",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "50",
+      id: "investor",
+    },
+    {
+      name: "Jordan Belfort's Cat",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "50",
+      id: "investor",
+    },
+    {
+      name: "Warren Buffett's Left Nut",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "130",
+      id: "investor",
+    },
+    {
+      name: "Warren Buffett's Blow",
+      indicators: ["RSI", "MACD", "OBV"],
+      recentPrice: "128.34",
+      percentChanged: "4.8",
+      open: "50",
+      id: "investor",
+    },
+  ];
+
+  return (
```

Comment:
Does this mean that if there are 100 jobs for example the web page will become extremely long because it shows all jobs at once? If that is the case we should look into only rendering about 6 at a time like it is now and making it scrollable to see other jobs

Reply:
Yeah ideally that would be the case. I wasn't sure how to implement that with dummy data, but I'll play around with it

Reply:
I think you would have to put the job list in a scrollable div and give the div a max height. Inside the div you would have to use the map function to render all the jobs.